src/patterns.rs

```diff
@@ -371,6 +371,8 @@ fn rewrite_tuple_pat(
         context.config.max_width(),
         if add_comma {
             Some(SeparatorTactic::Always)
+        } else if dotdot_pos.is_some() {
```

Comment:
Should this come before the `add_comma` clause? It seems that in the case that `add_comma && dotdot_pos.is_some()` we'd get `Always` here where previously we got `Never`.

Reply:
Yes I believe you are right. I will update this along with tidying up the git history. Thanks!